docs/LDAP.rst

```diff
@@ -21,61 +21,287 @@ plug to a directory.
+   |                                                | alias for com.sun.jndi.ldap.connect.pool.authentication |                           |                                    |                            |
+   +------------------------------------------------+---------------------------------------------------------+---------------------------+------------------------------------+----------------------------+
+   | exo.ldap.connection.pool.maxsize               | Maximum number of connections per connection            | Any positive number       | 300000                             |                            |
+   |                                                | identity that can be maintained concurrently,           |                           |                                    |                            |
+   |                                                | alias for com.sun.jndi.ldap.connect.pool.maxsize        |                           |                                    |                            |
+   +------------------------------------------------+---------------------------------------------------------+---------------------------+------------------------------------+----------------------------+
+   | exo.ldap.connection.pool.timeout               | Number of milliseconds that an idle connection          | Any non negative number   |                                    |                            |
+   |                                                | may remain in the pool without being closed             |                           |                                    |                            |
+   |                                                | and removed from the pool,                              |                           |                                    |                            |
+   |                                                | alias for com.sun.jndi.ldap.connect.pool.timeout        |                           |                                    |                            |
+   +------------------------------------------------+---------------------------------------------------------+---------------------------+------------------------------------+----------------------------+
+   | exo.ldap.connection.pool.protocol              | List of space-separated protocol types of               | ``plain`` and/or ``ssl``  | ``plain ssl``                      |                            |
+   |                                                | connections that may be pooled,                         |                           |                                    |                            |
+   |                                                | alias for com.sun.jndi.ldap.connect.pool.protocol       |                           |                                    |                            |
```

Comment:
it is not clear wether this property can be defined in the exo.properties onot what happens is tit's defined as a System property. Does exo.ldap.connection.pool.protocol overrides com.sun.jndi.ldap.connect.pool.protocol in that case or is it just ignored ?

Reply:
I don't know, and I don't find any doc about that, I will check in the source code and let you know.

Reply:
"exo.ldap.*" properties override the "com.sun.*" for [those defined in the picketlink configuration](https://github.com/Meeds-io/gatein-portal/pull/16/files#diff-6b70d8d0aaf06025d73f66871106c7f9R316-R322). After doing some tests, the behavior is quite confusing:
* there are 2 ways to define these variables
* variables overridden in picketlink configuration are set only when the first LDAP connection is initiated, which means that the value of com.sun.* properties can have a first value at startup, then another value after the first LDAP connection

I propose to remove these custom variables in PicketLink configuration. Administrators will simply customize LDAP connection pool by defining com.sun.* properties directly, either in exo.properties or in bin/setenv-customize.sh. And defaults values will be [JVM default values](https://docs.oracle.com/javase/jndi/tutorial/ldap/connect/config.html). We use the standards.
WDYT ?

Reply:
Done. Tell me if it is not ok for you.